### ISSUE TITLE
New Task Schedule - Purge orphans attributes and empty events

### ISSUE THREAD

**Issue by davecabio** [MISP/MISP#7252]
**Is your feature request related to a problem? Please describe.**
Currently to clean MISP db it's necessary to launch a cron script (or a service) that through pymisp delete emtpy events and orphans attributes.

**Describe the solution you'd like**
Create two new scheduled task that will allow an admin to schedule the orphans attribute and empty events purge.

**Comment by chrisinmtown** [comment]
I agree with the need, but apparently the MISP scheduler is utterly unreliable so I'm not sure that's the right way to proceed. We use crontab entries for database cleanup, like aging out old events.
